README.md

```diff
@@ -11,5 +11,5 @@ Use the [wiki](https://github.com/philippeller/freeDOM/wiki) for things other th
 
 It is recommended to install in-place, i.e.:
 ```
-pip install --editable .
+pip install --editable .[full]
 ```
```

Comment:
can we add a sentence about the new i3 module, how to install it and where to find the below example snippet?